### ISSUE TITLE
ui5-multi-input: [a11y] cannot trigger value-help-icon-press event via keyboard

### ISSUE THREAD

**Issue by zhangliangyi** [SAP/ui5-webcomponents#2143]
**Describe the bug**
When create a ui5-multi-input, I could not trigger the `value-help-icon-press` event via keyboard. I tried to use the `alt/option+arrow down` just like the way in `datepicker`, but nothing happened.

**Expected behavior**
The `value-help-icon-press` event should be triggered after user press `alt/option+arrow down` which is same as https://sapui5.hana.ondemand.com/#/entity/sap.m.Input/sample/sap.m.sample.InputAssisted

**Context**
- UI5 Web Components version: 0.23.1
- OS/Platform: MacOS
- Browser *(if relevant)*: Chrome
- Other information: N/A

**Affected components** *(if known)*
ui5-multi-input

Btw, is it necessary to add other aria attributes such as `aria-haspopup="listbox"` or provide an interface for component caller?

Thanks
